videostream.js

```diff
@@ -51,20 +52,14 @@ function VideoStream (file, mediaElem, opts) {
 	})
 }
 
-inherits(VideoStream, EventEmitter)
-
 VideoStream.prototype._pump = function () {
 	var self = this
 	self._tracks.forEach(function (track) {
-		pump(track.muxed, track.mediaSource, function (err) {
-			if (err && err.message !== 'premature close') {
```

Comment:
It looks like the `'premature close'` error originates from within the `pump` package (specifically, it's `end-of-stream` dependency). So because we removed the callback to `pump()`, this error should not surface.

In my first comment, I was under the impression that this was an error from the muxer or mediasource streams.